# 10. Interest Rate Futures

In [3]:
from fixedincome2025 import table

## Types of Interest Rate Futures

* Reference: Section 6.3 of Options, Futures, and Other Derivatives 11th Edition by John Hull, Sankarshan Basu
* Futures are exchange products
* There is a margin mechanism to avoid default risk
* In the U.S., by far the most pupolar interest rate futures are
    * Treasury futures
    * SOFR futures
* Both products of the CME group

## Treasury Futures Overview

* Historical note: Treasury futures were CBOT (Chicago Board of Trade) products. CBOT merged with CME (Chicago Mercantile Exchange) in 2007 to form the current CME group
* Contract size: One contract involves the delivery of $\$100,000$ face value of the bonds
* Settlement: Physical, meaning if you are long you actually get T-bonds on delivery date, not cash

* Delivery date
    * There is not a set delivery date. The party with a short position can choose to deliver on any business day of the contract month

## Deliverable Bonds

* A typical futures contract requires one party to deliver a specified quantity of the underlying commodity (think of [15,000 pounds of frozen concentrated orange juice](https://www.ice.com/products/30/FCOJ-A-Futures)) to the other on the delivery date
* But there are so many treasuries with various dates and coupon rates this won't work
* Treasury futures are structured in a way that, all *similar* treasuries can be delivered
    * But for a single futures contract you must deliver a single eligible bond; you cannot split one contract's delivery across two different bonds

## Quotes

* Treasury bond futures prices are quoted int he same way as the Treasury bond prices (the spot market)
* Quotes are in dollars and 32nds of a dollar per $\$100$ face value
    * $110$-$03$ means $\$110\frac{3}{32}$ for a T-bond with face value $\$100$
    * One futures contract represents $\$100,000$ face value of the delivered bonds so $1,000$ such bonds with face value $\$100$
    * If the quoted price dropded by $\$1$, say from $110$-$03$ to $109$-$03$, the holder of a futures contract lose $\$1,000$

## 6 Types of Treasury Futures

* What bonds are deliverable? 

In [14]:
table('treasury_futures')

Ticker Remaining Term Original Term  \
2-Year T-Note Futures               ZTZ5     [1y9m, 2y]       <= 5y3m   
5-Year T-Note Futures               ZFZ5        >= 4y2m       <= 5y3m   
10-Year T-Note Futures              ZNZ5     [6.5y, 8y]                 
Ultra 10-Year U.S. T-Note Futures   TNZ5    [9y5m, 10y]           10y   
U.S. T-Bond Futures                 ZBZ5     [15y, 25y)                 
Ultra U.S. T-Bond Futures           UBZ5         >= 25y                 

                                             Notes  
2-Year T-Note Futures                               
5-Year T-Note Futures                               
10-Year T-Note Futures                              
Ultra 10-Year U.S. T-Note Futures                   
U.S. T-Bond Futures                Can be callable  
Ultra U.S. T-Bond Futures

$$
$$
* The remaining term is counting from the first day of the contract delivery month Dec 2025
* For specs of Treasury futures, see [cmegroup.com](https://www.cmegroup.com/) > MARKETS > Interest Rates > U.S. Treasuries
    * More detailed spec in the [rule book](https://www.cmegroup.com/rulebook/CBOT/), for example [Chapter 18](https://www.cmegroup.com/rulebook/CBOT/II/18.pdf) for ZBZ5

## Delivery Month Code

* ZBZ5 means Dec 2025 contract, ZBH6 means Mar 2025 contract, etc.

$$
$$

| Code | Month | Code | Month | Code | Month |
|:-----|:------|:-----|:------|:-----|:------|
| F | January | G | February | H | March |
| J | April | K | May | M | June |
| N | July | Q | August | U | September |
| V | October | X | November | Z | December |

$$
$$

* There are **quarterly futures** for March, June, September and December (HMUZ) and **serial futures** for other "off-quarter" months
* For Treasury futures, only quarterly are trading. For SOFR futures, serial are also trading
* Quarterly are much more liquid and important

## Conversion Factor

* Because you can choose to deliver from a basket of bonds, it wouldn't make sense to settle all contracts with one single futures settlement price. It has to be adjusted
* Cash received by the short when a bond is delivered is
$$
(\text{Futures settlement} \times \text{Conversion factor}) + \text{Accrued interest}
$$
* Futures settlement is the most recent futures settlement price
* Roughly speaking, the conversion factor should be larger when the delivered bonds are more expensive

## Conversion Factor: Example

$$
$$
$$
(\text{Futures settlement} \times \text{Conversion factor}) + \text{Accrued interest}
$$
* Assuming
    * Settlement price is $90$-$00$
    * Conversion factor for the bond delivered is $1.38$
    * Accrued interest on this bond at the time of delivery is $\$3$ per $\$100$ face value
* Cash received by the party with the short position is
$$
(90.00\times 1.3800) + 3.00 = \$127.20
$$
per $\$100$ face value. Since one contract delivers bonds with a face value of $\$100,000$, overall cash received is $\$127,200$

## Computing the Conversion Factor

* The conversion factor for a bond is equal to the quoted price the bond would have per dollar of principal on the first day of the delivery month on the assumption that the interest rate for all maturities equals $6\%$ per annum (with semiannual compounding)

* The bond maturity and the times to the coupon payment dates are rounded down to the nearest 3 months

* If, after rounding, the bond lasts for an exact number of 6-month periods, the first coupon is assumed to be paid in 6 months

* If, after rounding, the bond does not last for an exact number of 6-month periods (i.e., there are an extra 3 months), the first coupon is assumed to be paid in 3 months and accrued interest is subtracted

## Computing the Conversion Factor: Example

* Consider a $10\%$ coupon bond with 20 years and 2 months to maturity
    * Each semiannual coupon payment is $5\%$
* First round down to 20 years
* Assuming face value is $\$100$, when the annualized discount rate is $6\%$ (or $3\%$ semiannually), the value of the bond is
$$
\sum_{j=1}^{40}\frac{5}{1.03^j} + \frac{100}{1.03^{40}} = \$146.23
$$
* Dividing by the face value gives a conversion factor of $1.4623$

## Cheapest-To-Deliver (CTD) Bond

* The conversion factors only roughly adjust the differences of deliverable bonds. There is still one that's the cheapest to deliver
* The party with a short position needs to deliver the bonds and receive cash
$$
(\text{Futures settlement} \times \text{Conversion factor}) + \text{Accrued interest}
$$

* The party can deliver the bonds it just purchased in the market at
$$
\text{Quoted bond price} + \text{Accrued interest}
$$

* The cost to do so is therefore
$$
\text{Quoted bond price} - (\text{Futures settlement} \times \text{Conversion factor})
$$

* The party should (and will) choose the bond that minimizes the cost to deliver

## CTD Bond: Example

* Assuming the most recent futures settlement price is $93$-$08$, or $93.25$ and we have 3 deliverable bonds as below: 

In [16]:
table('ctd_bond')

Quoted Price ($)  Conversion Factor
Bond 1             99.50             1.0382
Bond 2            143.50             1.5188
Bond 3            119.75             1.2615

* The cost of delivering each is
\begin{align*}
&\text{Bond 1: }\quad 99.5 - (93.25\times 1.0382) = \$2.69\\
&\text{Bond 2: }\quad 143.5 - (93.25\times 1.5188) = \$1.87\\
&\text{Bond 3: }\quad 119.75 - (93.25\times 1.2615) = \$2.12\\
\end{align*}
* The CTD bond is Bond 2